integration_tests/integration/offenceCodeSelection.spec.ts

```diff
@@ -1,6 +1,7 @@
 import OffenceCodeSelection from '../pages/offenceCodeSelection'
 import Page from '../pages/page'
 import DetailsOfOffence from '../pages/detailsOfOffence'
+import CheckYourAnswersPage from '../pages/taskList'
```

Comment:
Yikes - this is my bad, fallen foul of the copying-and-pasting to start a new test file and not renaming the class...

Reply:
Any chance you could rename it as part of this PR?